packages/hyperion-react-testapp/src/AutoLoggingWrapper.ts

```diff
@@ -53,6 +53,7 @@ export function init() {
     'al_network_request',
     'al_network_response',
     'al_network_response',
+    'al_flowlet_event',
```

Comment:
Can you see these emitted in the test app?  I checked out the branch but not seeing any.

Reply:
yes, I do see them. Did you do a full build after checking out? 

![image](https://github.com/facebook/hyperion/assets/3509019/380850b7-c09d-4737-8c98-ba24a8ea03e5)